blockchain/state/statedb.go

```diff
@@ -326,6 +326,20 @@ func (self *StateDB) IsValidCodeFormat(addr common.Address) bool {
 	return false
 }
 
+// GetCodeFormat can return an invalid codeFormat, params.CodeFormatLast.
```

Comment:
This sentence sounds a bit confusing. What did you intend?

Reply:
I wrote the comment to tell that this function can return invalid codeFormat if `getStateObject(addr)` or `GetProgramAccount(stateObject.account)` is failed. please take a look https://github.com/klaytn/klaytn/commit/b62138adad3b89433c763458df317a578be8d84a